tests/security/cc/crypto.pm

```diff
@@ -22,9 +22,6 @@ sub run {
     # Install certification-sles-eal4: needed by test case `crypto`
     zypper_call('in certification-sles-eal4');
 
-    # Export MODE
-    assert_script_run("export MODE=$audit_test::mode");
-
```

Comment:
It will only affect this case, right?

Reply:
yes, we have done `export MODE=$audit_test::mode` in `run_testcase`, so we could remove this line.